lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -732,8 +732,11 @@ def get(self, request, *args, **kwargs):
             cilset = CourseImportLog.objects.filter(course_id=course_id).order_by('-created')
             log.debug('cilset length={0}'.format(len(cilset)))
         mdb.disconnect()
-        context = {'cilset': cilset,
-                   'course_id': course_id.to_deprecated_string() if course_id else None,
-                   'error_msg': error_msg}
+        context = {
+            'cilset': cilset,
+            'course_id': course_id.to_deprecated_string() if course_id else None,
+            'detailed_view': course_id is not None,
```

Comment:
Removed the "readability" commit. Reading this template was a little bit confusing at first with it checking if course_id isn't `None` all over the place, but I can see how this change doesn't really make it much more explicit.